GitCommands/AppTitleGenerator.cs

```diff
@@ -20,17 +22,18 @@ public interface IAppTitleGenerator
     /// <summary>
     /// Generates application title.
     /// </summary>
+    [Export(typeof(IAppTitleGenerator))]
     public sealed class AppTitleGenerator : IAppTitleGenerator
     {
+        private readonly IRepositoryDescriptionProvider _descriptionProvider;
 #if DEBUG
         private static string? _extraInfo;
 #endif
 
-        private readonly IRepositoryDescriptionProvider _description;
-
-        public AppTitleGenerator(IRepositoryDescriptionProvider description)
+        [ImportingConstructor]
+        public AppTitleGenerator(IRepositoryDescriptionProvider descriptionProvider)
         {
-            _description = description;
+            _descriptionProvider = descriptionProvider ?? throw new ArgumentNullException(nameof(descriptionProvider));
```

Comment:
💡 This null check is not necessary. MEF will not instantiate this part if a non-null instance of `IRepositoryDescriptionProvider` is not available.